cad/fp-info-cache

```diff
@@ -0,0 +1 @@
+0
```

Comment:
Was this added by accident?

Reply:
Likely automatic EOF conversion by git | Windows ⬌ Linux

Reply:
Can you revert it in that case?

Reply:
Oh right, no that file is just generated by KiCAD, you will probably want to add it to the `.gitignore` file then.